qa/vagrant/src/test/resources/packaging/tests/99_upgrade_repos.bats

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bats
+
+# Tests upgrading elasticsearch from a previous version with the deb or rpm
+# packages. Just uses a single node cluster on the current machine rather than
```

Comment:
I'd leave a comment about how this doesn't use the Elasticsearch version in the source tree - just the upgrade from version and the configured repository.